pom.xml

```diff
@@ -216,13 +221,20 @@
 		<!-- SECOM -->
 		<dependency>
 			<groupId>org.grad.secom</groupId>
-			<artifactId>secom-core</artifactId>
-			<version>0.0.11</version>
+			<artifactId>secom-core-jakarta</artifactId>
+			<version>0.0.12</version>
 		</dependency>
 		<dependency>
 			<groupId>org.grad.secom</groupId>
-			<artifactId>secom-springboot</artifactId>
-			<version>0.0.11</version>
+			<artifactId>secom-springboot3</artifactId>
+			<version>0.0.12</version>
+		</dependency>
+
+		<!-- G1128  Schemas -->
+		<dependency>
+			<groupId>net.maritimeconnectivity.eNav</groupId>
+			<artifactId>g1128-schemas-jakarta</artifactId>
+			<version>0.1.4</version>
 		</dependency>
```

Comment:
Should we update the MCC fork of this dependency?
Also, instead of having different artifactIds for the JavaEE and JakartaEE version, would it make sense / be possible to have different classifiers like they do here https://github.com/haraldk/TwelveMonkeys/blob/master/servlet/pom.xml?

Reply:
What a nice idea!

I've updated the G-1128 Schemas repo: https://github.com/gla-rad/G1128-Schemas

Now we can use the dependancy using the classifier... wasn't aware of this module... many thanks!

Reply:
I've also updated a bit the README to be  more up-to-date

Reply:
That was quick! Nicely done! 😄 
Do you have any last-minute changes you want to do, because else I think it looks fine to be merged?

Reply:
Awesome! Then I will merge it! Great work again! 😄